services/painting-process-equipment-defect-detection-model-service/app/models/model_config.yaml

```
@@ -0,0 +1,40 @@
+    machine_issue: "PAINT-EQ-MCH"
+    voltage_issue: "PAINT-EQ-VOL"
+    current_issue: "PAINT-EQ-CUR"
+    temperature_issue: "PAINT-EQ-TEM"
+  # 로그 파일 저장 경로 수정 (선택 사항, 현재 경로 유지)
+  file_path: "logs/issue_logs.jsonl" # 이 경로는 서비스 루트 기준
+
+validation:
+  voltage_range: [20, 350]
+  current_range: [5, 400]
+  temperature_range: [25, 40]
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)